test/EVMHost.cpp

```diff
@@ -754,3 +756,20 @@ evmc::result EVMHost::resultWithGas(
 	result.output_size = _data.size();
 	return result;
 }
+
+void EVMHost::print_all_storage(ostringstream& _os)
+{
+	for (auto const& [addr, mockedAccount]: accounts)
+	{
+		_os << "Address: " << convertFromEVMC(addr) << endl;
+		for (auto const& [slot, value]: get_address_storage(addr))
+			if (get_storage(addr, slot))
+				_os << convertFromEVMC(slot) << ": " << convertFromEVMC(value.value) << endl;
+	}
+}
```

Comment:
Also for this one I'm not *entirely* sure, if it belongs in the ``EVMHost`` or whether it should just return a const ref to ``accounts`` and the caller side should do the printing, etc.
But fine either way!

Reply:
Right. I am actually not sure what belongs in the host so I a placed this here for now. Wondering if the host is the right place for "tracing" features now that we may actually have to code in more tracing logic. An alternative could be some thin interface between client (fuzzer) and server (EVMHost) if one can visualise them that way. Wdyt?

Reply:
Yeah, I'm not entirely sure either, especially since this uses the ``convertFromEVMC`` functions and all that... maybe it's just fine to leave it like this here for now.